src/monitor/commands/get-monitor.command.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/database/prisma.service';
+import { Monitor } from '../domain/monitor';
+import { MonitorNotFoundException } from 'src/student/utils/exceptions';
+import { MonitorFactory } from '../utils/monitor.factory';
+
+@Injectable()
+export class GetMonitorCommand {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  async execute(monitorId: number): Promise<Monitor> {
+    const monitor = await this.prismaService.monitor.findUnique({
+      where: {
+        id: monitorId,
+      },
+      include: {
+        responsible_professor: { include: { user: true } },
+        student: { include: { user: true } },
+        subject: { include: { course: true } },
+        AvailableTimes: true,
+        status: true,
+        MonitorSettings: true,
```

Comment:
faz assim pra trazer só o válido e não todos:

```suggestion
        MonitorSettings: {
          select: {
            id: true,
            preferencial_place: true,
            is_active: true,
          },
          where: {
            is_active: true,
          }
        },
```